Tribler/community/chant/MDPackXDR.py

```diff
@@ -0,0 +1,68 @@
+import xdrlib
+from Tribler.community.chant.timeutils import time2float, float2time
+from Tribler.pyipv8.ipv8.keyvault.crypto import ECCrypto
+
+INFOHASH_SIZE = 20 # bytes
+REGULAR_TORRENT = 1
+CHANNEL_TORRENT = 2 # version
+MD_DELETE = 3
+
+crypto = ECCrypto()
+
+def serialize_metadata_gossip(md, key=None):
+    p = xdrlib.Packer()
+
+    p.pack_int(                   md["type"])
```

Comment:
This whitespace formatting is not PEP8 compliant.

Reply:
Please stick to PEP8 compliance, it is the standard we have agreed on and what all of our style checkers use. Diverging from this also means that everyone who touches this in the future will get the same errors.